src/ngraph/runtime/backend.hpp

```diff
@@ -109,4 +115,11 @@ class ngraph::runtime::Backend
     virtual bool is_supported_property(const Property prop) const;
 
     virtual void remove_compiled_function(std::shared_ptr<Executable> exec);
+
+    virtual std::shared_ptr<ngraph::runtime::Allocator> get_framework_memory_allocator();
+    virtual void set_framework_memory_allocator(
+        const std::shared_ptr<ngraph::runtime::Allocator>& allocator);
+    virtual ngraph::runtime::AllocateFunc get_device_memory_alloc();
+    virtual ngraph::runtime::DestroyFunc get_device_memory_dealloc();
+    virtual bool is_device_memory();
```

Comment:
Some documentation for these methods would also be great. Not sure what `is_device_memory` means exactly.